lib/gems/pending/util/vmdb-logger.rb

```diff
@@ -156,8 +157,9 @@ class Formatter < Logger::Formatter
     FORMAT = "[----] %s, [%s#%d:%x] %5s -- %s: %s\n"
 
     def call(severity, time, progname, msg)
-      msg = prefix_task_id(msg2str(msg))
+      max_log_line_length = 1.megabyte
```

Comment:
Since this call will happen very frequently, I'm thinking this might be better moved to a constant.  That way it doesn't have to re-evaluate 1.megabyte over and over.

Reply:
Also, is there an active_support/core_ext for this megayute method?

Reply:
Good point it is.